package.json

```diff
@@ -20,7 +20,7 @@
     "@bcrs-shared-components/contact-info": "1.1.2",
     "@bcrs-shared-components/corp-type-module": "1.0.6",
     "@bcrs-shared-components/court-order-poa": "2.0.0",
-    "@bcrs-shared-components/date-picker": "1.1.12",
+    "@bcrs-shared-components/date-picker": "1.1.15",
```

Comment:
The actual fix is in the shared component. If you're curious, see https://github.com/bcgov/bcrs-shared-components/pull/129

Reply:
Proof of fix: note that `pointer-events` has been overridden to "auto"...

![image](https://user-images.githubusercontent.com/10002889/167534388-5a41a786-dc21-419d-9f0b-aad7810980f1.png)